lib/WUI/nhttp/status_page.cpp

```diff
@@ -23,12 +23,19 @@ Step StatusPage::step(std::string_view, bool, uint8_t *output, size_t output_siz
     if (status == Status::NoContent) {
         content_buffer[0] = 0;
     } else {
-        snprintf(content_buffer, sizeof(content_buffer), "<html><body><h1>%u: %s</h1><p>%s", static_cast<unsigned>(status), text.text, extra_content);
+        snprintf(content_buffer, sizeof(content_buffer), "%u: %s\n\n%s\n", static_cast<unsigned>(status), text.text, extra_content);
```

Comment:
nitpick: printf format macro :wink: ? Something like PRIu8 or similar based on `status`?

Reply:
Well, `status` is of type `Status` and there's no `PRI_webstatus` thing around. It does have the values set to the right numeric values (here it's one place where it IMO actually makes sense).

Anyway, I need to cast it to _something_ that's printable and in that case it as well may be `unsigned` as well as anything else.